subprojects/maven/src/main/java/org/gradle/api/artifacts/maven/MavenPom.java

```diff
@@ -58,6 +59,17 @@
     MavenPom project(Closure pom);
 
     /**
+     * Provides a builder for the Maven POM for adding or modifying properties of the Maven {@link #getModel()}.
+     * The syntax is exactly the same as used by polyglot Maven.
+     *
+     * @return this
+     *
+     * @see #project(Closure)
```

Comment:
Shouldn't we document the `Action` method primarily, and link to it from the `Closure` variant?